src/Sulu/Bundle/ContentBundle/Tests/Features/content_type_url.feature

```diff
@@ -6,7 +6,7 @@ Feature: Url content type
     Background:
         Given there exists a page template "url_page" with the following property configuration
         """
-        <property name="url" type="url">
+        <property name="this_url" type="url">
```

Comment:
Hmm. This was strange.

Basically, the system doesn't seem to care if there are two properties with the same name, both here and in `develop`. The difference is that now, the second will replace the first, and the second does not have the `sulu.rlp` flag.